iso15118/secc/controller/evse_data.py

```diff
@@ -0,0 +1,126 @@
+from dataclasses import dataclass
+from typing import Optional, Union
+
+from iso15118.secc.controller.common import Limits
+
+
+@dataclass
+class EVSEACCLLimits(Limits):
+    # Optional in both Scheduled and Dynamic CL (both AC CL and BPT AC CL)
+    evse_target_active_power: Optional[float] = None  # Required in Dynamic AC CL
+    evse_target_active_power_l2: Optional[float] = None
+    evse_target_active_power_l3: Optional[float] = None
+    evse_target_reactive_power: Optional[float] = None
+    evse_target_reactive_power_l2: Optional[float] = None
+    evse_target_reactive_power_l3: Optional[float] = None
+    evse_present_active_power: Optional[float] = None  # Optional in AC CPD
+    evse_present_active_power_l2: Optional[float] = None  # Optional in AC CPD
+    evse_present_active_power_l3: Optional[float] = None  # Optional in AC CPD
+
+
+@dataclass
+class EVSEACLimits(Limits):
```

Comment:
can we change the name to `EVSEACCPDLimits` and add a description to the class expanding the name

Reply:
Sure, done.